### PR TITLE
created an events model for models.py

### PR DESCRIPTION
**Pull request by papycoda**

### CODE REVIEW

**Review by danieliheonu — COMMENTED** [review]
why do we have an event model

**Comment by papycoda** [comment]
every notice can be referred to as an event

File: backend/noticeboard/notice/models.py [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
 # Create your models here.
+class Event(models.Model):
+  Priority_Choices = [
+        (Very_Urgent, 'Very Urgent'),
+        (Urgent, 'Urgent'),
+        (Fairly_urgent, 'Fairly Urgent'),
+        (Regular, 'Regular'),
+        ]
+	event_name = models.CharField(max_length = 75)
+  event_type = models.CharField(max_length = 75)
```
Comment (danieliheonu) [inline backend/noticeboard/notice/models.py:12]: what are the types the user can have

File: backend/noticeboard/notice/models.py [hunk 2]
```diff
@@ -1,3 +1,21 @@
...
+  event_priority = models.Charfield(max length =80,choices = priority_choices,default = "Regular")
+	event_description = models.CharField(max_length = 500)
+	event_date = models.DateField()
+  event_timings = models.TimeField()
+	event_venue = models.CharField(max_length = 75)
```
Comment (danieliheonu) [inline backend/noticeboard/notice/models.py:17]: why is there an event venue?

File: backend/noticeboard/notice/models.py [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+	event_organizer_details = models.CharField(max_length= 140)
```
Comment (danieliheonu) [inline backend/noticeboard/notice/models.py:18]: why do you have an organizer detail? why don't you just make this user_id which is a foreign key to an account table?

**Review by danieliheonu — COMMENTED** [review]
your model doesn't even have an id field

**Review by danieliheonu — APPROVED** [review]

**Pull request merged**